coordinator/src/node/closed_positions.rs

```diff
@@ -0,0 +1,37 @@
+use crate::db;
+use crate::node::Node;
+use anyhow::Context;
+use anyhow::Result;
+use bitcoin::hashes::hex::ToHex;
+
+pub fn sync(node: Node) -> Result<()> {
+    let mut conn = node.pool.get()?;
+    let open_and_closing_positions =
+        db::positions::Position::get_all_open_or_closing_positions(&mut conn)
+            .context("Failed to load open and closing positions")?;
+
+    for position in open_and_closing_positions {
+        let contract = match node
+            .inner
+            .get_closed_contract(position.temporary_contract_id)
+        {
+            Ok(Some(closed_contract)) => closed_contract,
+            Ok(None) => {
+                tracing::trace!(position_id=%position.id, "Position not closed yet, skipping");
```

Comment:
🔧 we might want to add a timeout here and set the position to failed if its still in closing. No need to repeat that sync if the position is in closing for days.

Reply:
I agree that this feature might be good to have, but I think this should be solved outside of this PR.